index.js

```diff
@@ -76,6 +76,14 @@ module.exports = function Expirer(timeoutMs, db, checkIntervalMs) {
 
 	expirer.touch = expirer.emit.bind(expirer, 'touch')
 	expirer.forget = expirer.emit.bind(expirer, 'forget')
+	expirer.createIfNotExists = function createIfNotExists(key) {
+		db.get(key, function (err, value) {
+			if (err && err.notFound) {
+				console.log('creating ' + key)
+				expirer.emit('touch', key)
```

Comment:
Added note in https://github.com/TehShrike/expire-unused-keys/pull/9/commits/529c95e89a04992f946a38ea8811175a53ab7a8d.